tests/integration/test_integration.py

```diff
@@ -0,0 +1,26 @@
+import pytest
+
+from unittest import mock
+
+from integration.integration_snowpark import snowpark_session
+
+
+@pytest.mark.skip(reason="not ready yet")
+@pytest.mark.integration
```

Comment:
That's fine. However, maybe we can just exclude all integration tests?

Reply:
Integration tests are excluded by default

```
[tool.pytest.ini_options]
addopts = "-m 'not integration'"
```

I removed this parameter from tox